src/DGmethods/DGmodel.jl

```diff
@@ -433,9 +547,9 @@ function indefinite_stack_integral!(
     FT = eltype(Q)
 
     # do integrals
-    nelem = length(topology.elems)
+    nelem = length(elems)
     nvertelem = topology.stacksize
-    nhorzelem = div(nelem, nvertelem)
+    horzelems = fld1(first(elems), nvertelem):fld1(last(elems), nvertelem)
```

Comment:
Ah, this is a pattern that's been copied in a number of places and might be worth `grep`ping for to change in all...

Reply:
There are corresponding changes that need to be applied to the kernels to match these changes.  I would instead suggest (in a different pr) remove the `elems` argument from kernels that don't use them.